utils/Makefile

```diff
@@ -0,0 +1,64 @@
+#		Tempesta FW
+#
+# Copyright (C) 2024 Tempesta Technologies, Inc.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License,
+# or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 59
+# Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+
+CLICKHOUSE_DIR := clickhouse-cpp
+CLICKHOUSE_BUILD_DIR := $(CLICKHOUSE_DIR)/build
+CLICKHOUSE_REPO_URL := https://github.com/ClickHouse/clickhouse-cpp.git
+
+CXX := g++
+LDFLAGS := -lfmt -lboost_program_options
+LDFLAGS += -lclickhouse-cpp-lib -L $(CLICKHOUSE_BUILD_DIR)/clickhouse
+LDFLAGS += -lcityhash -L $(CLICKHOUSE_BUILD_DIR)/contrib/cityhash/cityhash
+LDFLAGS += -lzstd -L $(CLICKHOUSE_BUILD_DIR)/contrib/zstd/zstd
```

Comment:
-lzstdstatic, because currently only static library was build. Or we should build shared library

Reply:
Fixed.